src/Core/AndroidNative/maui/build.gradle

```diff
@@ -28,6 +28,7 @@ android {
 dependencies {
     implementation 'com.github.bumptech.glide:glide:4.15.1'
     implementation 'com.github.bumptech.glide:annotations:4.15.1'
+    implementation 'com.github.bumptech.glide:avif-integration:4.15.1'
```

Comment:
We have this in Java, but how do we get this dependency in C#?

If we just have this:

https://github.com/dotnet/maui/blob/b6ea3df9318bc035fd23553faff765df0cf3b406/src/Core/src/Core.csproj#L50

Is there a second, new NuGet package?

Reply:
Yes there is new NuGet package


I worked on Sandbox sample 

and added to sample `csproj`:

```xml
  <ItemGroup Condition="$(TargetFramework.Contains('-android'))">
    <PackageReference Include="Xamarin.Android.Glide.AVIF.Integration" Version="4.16.0" />
  </ItemGroup>
```

IMO aading to sample should help for repro and fix testing.

I planned to add 

```xml
  <ItemGroup Condition="$(TargetFramework.Contains('-android'))">
    <PackageReference Include="Xamarin.Android.Glide.AVIF.Integration" Version="4.16.0" />
  </ItemGroup>
```

to `maui/src/Core/src/Core.csproj` before submitting for review.

Reply:
Seems like `main`/`net8.0` will need another Android Ecosystem updates or this will have to wait release of `net9.0`.

Reply:
Main is .NET 9 now, do you want to rebase this?

Reply:
Sure. I have to